### PR TITLE
Feat:Homepage

### PR DESCRIPTION
**Pull request by shubhamkumar648**
1.header
2. product category 
3. footer
home1
hom2

### CODE REVIEW

**Review by therajatg — APPROVED** [review]
looks clean

File: stylesheets/main.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;1,100&family=Roboto:wght@300&display=swap');
+@import url("./utils.css");
+@import url("./footer.css");
+@import url("./homepage.css");
+
+
+body {
```
Comment (krishanu2911) [inline stylesheets/main.css]: just a suggestion you may move this property to the utils.

**Review by krishanu2911 — APPROVED** [review]
Everything is good to go just added a suggestion well written code and nice design.

File: stylesheets/main.css (hunk 1 above)
Comment (shubhamkumar648) [reply stylesheets/main.css]: thanks for your aknowledge! i will do

**Comment by shubhamkumar648** [comment]
> looks clean

thank you

**Pull request merged**